gh-actions/go/gotestfmt/action.yaml

```diff
@@ -0,0 +1,23 @@
+name: Install gotestfmt and our wrapper script
+description: Install gotestfmt and our wrapper script which improves the output and copies it to log files
+
+runs:
+  using: "composite"
+  steps:
+    - name: Install gotestfmt
+      run: go install github.com/gotesttools/gotestfmt/v2/cmd/gotestfmt@latest
+      shell: bash
+
+    - name: Checkout repository
+      uses: actions/checkout@v4
```

Comment:
Oh, good that you're asking! This checks out the repo in `GITHUB_WORKSPACE` which is a different directory than `GITHUB_ACTION_PATH`. So turns out that we don't need to use the `checkout` action, because the repo is already checked out in `GITHUB_ACTION_PATH`.

Reply:
ah, they changed this behaviour! In the past, only the composite action file itself was available, but not the other files from the repo. I guess most people were doing this checkout :) Glad that I asked! :)